### PR TITLE
Fix Sphinx errors in box_nms

### PR DESCRIPTION
**Pull request by vandanavk**
## Description ##
Fixes https://github.com/apache/incubator-mxnet/issues/13016, Fixes https://github.com/apache/incubator-mxnet/issues/13017

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- Fix indentation

### CODE REVIEW

**Comment by vandanavk** [comment]
Preview:
http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13261/1/api/python/ndarray/contrib.html

http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13261/1/api/python/symbol/contrib.html

**Comment by kalyc** [comment]
@nswamy @aaronmarkham could you review the PR?
@mxnet-label-bot add [pr-awaiting-merge]

File: src/operator/contrib/bounding_box.cc [hunk 1]
```diff
@@ -47,23 +47,25 @@ the in_grad will be assigned 0.
...
-- `coord_start`: required, default=2, the starting index of the 4 coordinates.
-Two formats are supported:
-  `corner`: [xmin, ymin, xmax, ymax]
-  `center`: [x, y, width, height]
-- `score_index`: required, default=1, box score/confidence.
-When two boxes overlap IOU > `overlap_thresh`, the one with smaller score will be suppressed.
-- `in_format` and `out_format`: default='corner', specify in/out box formats.
+additional elements are allowed::
+
+  - `id_index`: optional, use -1 to ignore, useful if `force_suppress=False`, which means
```
Comment (aaronmarkham) [inline src/operator/contrib/bounding_box.cc]: I think maybe this should not be indented.
If you look at the preview it throws the text in a code block and the backticks don't render like they should.
http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13261/1/api/python/ndarray/contrib.html#mxnet.ndarray.contrib.box_nms

**Comment by vandanavk** [comment]
Updated preview: http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13261/4/api/python/ndarray/contrib.html

@aaronmarkham

**Comment by kalyc** [comment]
@mxnet-label-bot add [pr-awaiting-review]

**Comment by kalyc** [comment]
@mxnet-label-bot update [pr-awaiting-merge]

**Comment by aaronmarkham** [comment]
@mxnet-label-bot add [Doc]

**Pull request merged**